### PR TITLE
header and footer components completed along with adding a contact page

### PR DESCRIPTION
**Pull request by Mamba240824**

### CODE REVIEW

File: src/pages/Contact.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const Contact = () => {
+  return (
+    <>
+
```
Comment (sjproctor) [inline src/pages/Contact.js]: This component should return something.

File: src/components/Footer.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
-        </>
-    )
-
-}
-
+        <footer className="footer">
+          <div className="container">
+            <div className="row">
+              <div className="col-md-4">
+                <h1>The TrapHouse</h1>
```
Comment (sjproctor) [inline src/components/Footer.js]: The h1 tag is used for a header and is odd, semantically, to have in the footer. Can you change it to something else? A paragraph tag would work.

**Review by sjproctor — COMMENTED** [review]
Overall this looks good. Just a few things to adjust.

File: src/components/Footer.js (hunk 2 above)
Comment (Sakaa4Short) [reply src/components/Footer.js]: Swapped h1 for h2 for style optimization.

File: src/pages/Contact.js (hunk 1 above)
Comment (Sakaa4Short) [reply src/pages/Contact.js]: Added a line for the return to render something on the page.

**Review by sjproctor — APPROVED** [review]

**Pull request merged**